CefSharp/JavascriptException.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright © 2010-2017 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System.Runtime.Serialization;
+
+namespace CefSharp
+{
+    [DataContract]
```

Comment:
Unless there is a reason `DataContract` and `DataMember` should be removed.

Reply:
Just followed the same pattern that JavascriptResponse has

Reply:
JavaScriptResponse is used by the WCF service, hence the attributes.

Reply:
Do you have time to remove the attributes? Can be merged when that's done.

Reply:
Done